test.sh

```diff
@@ -62,6 +62,24 @@ do
   cat $test_dir/ci_results >> ci_results
 done
 
+if [[ -d "test_daemon_disabled" ]]; then
+  echo 'Check | Gold time (s) | PR time (s) | Delta (PR time - Gold time)' > results.markdown
+  echo '------|---------------|-------------|----------------------------' >> results.markdown
+  checks_in_pr=`jq '.Average | keys | .[]' test_daemon_disabled/time_tracker.json`
+  checks_in_master=`jq '.Average | keys | .[]' time_tracker.json`
+  for check in $checks_in_pr; do
+    pr_time=$(jq -r ".Average[$check]" test_daemon_disabled/time_tracker.json);
+    if [[ `echo $checks_in_master | grep -w $check` ]];
+    then
+      gold_time=$(jq -r ".Average[$check]" time_tracker.json);
+      delta=$(bc -l <<<"scale=6; (${pr_time}-${gold_time})/1")
```

Comment:
Let's display the delta in percentage i.e percentage time increase/decrease when compared to the master branch.